aspnetcore/fundamentals/openapi/customize-openapi.md

```diff
@@ -119,6 +119,22 @@ For example, the following schema transformer sets the `format` of decimal types
 
 [!code-csharp[](~/fundamentals/openapi/samples/10.x/WebMinOpenApi/Program.cs?name=snippet_schematransformer1)]
 
+## Support for generating OpenApiSchemas in transformers
+<!-- https://github.com/dotnet/aspnetcore/pull/61050 -->
```

Comment:
@wadepickett @guardrex 

Do we want to standardize on location and format for these links (which generally point to the PR that established the feature)?  We generally run across them directly below the relevant section heading. I have been putting them at the end of the section as `For more information see`... links

Reply:
We could.  I felt like it wasn't as valuable for the reader as it was for us to be able to go back and verify the original intent and changes.  Often those will contain info that becomes outdated later in the same conversation.  So it felt like it might cause more confusion.  However, I find it helpful to have available for us as authors to remember what prompted our update and who was involved if we need to back and ask questions.   The short answer: I don't vote to list these on the live page as links as a standard process.  I do think they are valuable as a hidden comment for the authors as we validate and update.

Reply:
I usually can find something useful in those PRs, but I understand the concern about parts becoming outdated. So leaving them as comments visible just to authors makes sense to me. In that case the location at the top of the section under the heading also makes sense to me.